packages/itwinui-css/package.json

```diff
@@ -3,19 +3,16 @@
   "version": "0.63.1",
   "author": "Bentley Systems",
   "license": "MIT",
-  "main": "src/index.scss",
   "files": [
     "css",
-    "scss",
     "CHANGELOG.md",
     "LICENSE.md"
   ],
-  "description": "CSS (Sass/SCSS) library for building beautiful and well working web UI components within Bentley Systems & iTwin.js applications.",
+  "main": "css/all.css",
+  "description": "CSS library for building beautiful and well working web UI components within Bentley Systems & iTwin.js applications.",
```

Comment:
Not related to this change: Are we still targeting only Bentley Systems & iTwin.js applications?

Reply:
I _think_ it's still accurate? Not sure who we are targeting tbh 😅